### DISCUSSION TITLE
RTE functionality

### DISCUSSION THREAD

**Discussion by Abhisheknanda1344463** [GrapesJS/grapesjs#3790] [Q&A]
Hi @artf I have implemented the new button on the toolbar i want to execute the RTE bold from that button.
How could i achieve that.
I've tried this ```rte.exec('bold')``` from the docs but it's not working.

Can you please help?
<img width="1434" alt="Screenshot 2021-09-18 at 3 28 50 PM" src="https://user-images.githubusercontent.com/20657737/133885142-be889f93-1028-4d1a-8109-7f43e63077d2.png">

**Comment by edwinjoel1984** [comment]
@Abhisheknanda1344463 can you share your code ?

**Reply by edwinjoel1984** [reply]
You can use rte.add
